### ISSUE TITLE
Analyze neuroblastoma crunch

### ISSUE THREAD

**Issue by jaclyn-taroni** [AlexsLemonade/refinebio#230]
### Context

After #182 is completed, we'll have [10+ neuroblastoma datasets](https://github.com/AlexsLemonade/refinebio/issues/112#issuecomment-363810177) that have been processed through refine.bio. _Part_ of our rationale for running these datasets prior to larger milestones (e.g., #227 and #223) is that working with this smaller set of samples will give us an opportunity to verify that we're producing useful output.

### Problem or idea

Some thoughts about cursory analyses:
* Quite a few of these datasets are available on [R2](https://hgserver1.amc.nl/cgi-bin/r2/main.cgi?&dscope=NRBL) and they're processed a number of ways (e.g., MAS5.0 seems to be pretty popular but sometimes something from the RMA "family" is used). We would not expect the values in data processed by refine.bio (processed with `SCAN.UPC::SCANfast`) to be wildly different from what is available on R2, so looking at correlation would be a nice check.
* Try a few aggregation strategies followed by visualization. I'll note that there some biologically relevant factors -- cell line v. primary tumor, presence of normal tissues, _MYCN_-amp, structural variants, staging -- that we should keep in mind.
* There is scientific interest in what pathways are predictive of survival; we could use something like GSVA to "summarize" pathway information and use those scores as features for survival analyses (similar to [Varn, et al. _BMC Medical Genomics._ 2015.](https://doi.org/10.1186/s12920-015-0086-0)).

### Solution or next step

Once #182 is completed (blocker), we'll get started!

### New Issue Checklist

- [x] The title is short and descriptive.
- [x] You have explained the context that led you to write this issue.
- [x] You have reported a problem or idea.
- [x] You have proposed a solution or next step.
